docs/dev_guide/frequently-asked-questions-faq.rst

```diff
@@ -343,3 +343,14 @@ is adjusted to avoid a bit shift that is greater than the type length:
        val = in[tid] << 32 - bit;
      ...
    }
+
+**How do I resolve missing include errors that occur when migrating my code?**
+
+Use the option ``--extra-arg=-v`` to prompt |tool_name| to use verbose
+output, which includes information about which paths the tool searches
+for includes. To add an additional path for the tool to use when
+searching for includes during migration, use the option
+``--extra-arg="-I<extra include path>"`` in your migration command.
+Alternatively, if you are using a compilation database, then you can
+add ``-I<extra include path>`` to the compile command for the
+necessary files.

```

Comment:
Suggest replacing the entire answer block with this updated version: 

Use the option ``--extra-arg=-v`` to prompt |tool_name| to use verbose
output, which includes information about which paths the tool searches
for includes.

You can provide an additional path to look for includes in one of the following
ways:

* Use the ``--extra-arg="-I<extra include path>"`` option in your migration command
  to specify an additional path for the tool to use when searching for includes
  during migration.

* If you are using a compilation database, add the ``-I<extra include path>``
  option to the compile command in the database for the source files, to 
  specify the include path.